indico/modules/events/management/templates/clone_event.html

```diff
@@ -0,0 +1,107 @@
+{% from 'forms/_form.html' import form_header, form_fieldset, form_footer, form_row, form_rows %}
+{% from 'message_box.html' import message_box %}
+
+{% macro invisible_field(field) -%}
+    {% set field_value = field._value() if field._value else field.data %}
+    {% set data = [field_value] if field_value is string else ([field_value|tojson] if field_value is mapping else field_value) %}
+    {% for value in data %}
+        <input type="hidden" name="{{ field.name }}" value="{{ value | forceescape }}">
+    {% endfor %}
+{%- endmacro %}
+
+{% macro invisible_fields(form, skip) %}
+    {% for field in form %}
```

Comment:
`{% for field in form if field.id not in skip %}`

Also, I'd use `field.short_name` even though it's usually the same as the ID (but wouldn't be if the form had a prefix set)